test/util/validator.test.js

```diff
@@ -0,0 +1,54 @@
+const { validateInputJson } = require('../../src/util/validator')
+
+test('validateInputJson', () => {
+  const goodRequest = {
```

Comment:
Rename this to `request`, and make it into its own test that passes successful validation.